libs/ui/src/components/CommentForm/CommentForm.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  Avatar,
+  Button,
+  HStack,
+  IconButton,
+  Stack,
+  Text,
+  Textarea,
+  useBreakpointValue,
+  VStack,
+} from '@chakra-ui/react'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { useTranslation } from 'next-i18next'
+import { useForm } from 'react-hook-form'
+import { FiArrowRight } from 'react-icons/fi'
+import * as yup from 'yup'
+
+import { FormItem } from '../FormItem'
+import { CommentFormProps } from './types'
+
+import { mutation } from '~lib'
+import { toastMessage } from '~utils'
+
+const userSchema = (t: TFunction<'translation', undefined>) =>

```

Comment:
```suggestion
const userSchema = (t: TFunction) =>
```

You probably forgot to import TFunction

`import { TFunction } from 'next-i18next'`

We only have `common` as translation files. It could be `TFunction<`common`>` if you import from `react-i18next` or only `TFunction`